src/types/schema.ts

```diff
@@ -0,0 +1,7 @@
+import { z } from "zod"
+
+import { FieldZ } from "./airtableFields"
+
+export const SchemaZ = z.record(FieldZ)
+// N.B. this is a type for a zod
+export type SchemaZodT = z.ZodType<z.infer<typeof SchemaZ>>
```

Comment:
Oh this is a a wayyyyyy better way of defining the TS type for Zod with a base chemae